pom.xml

```diff
@@ -102,20 +125,22 @@
 			<groupId>ch.qos.logback</groupId>
 			<artifactId>logback-classic</artifactId>
 			<version>1.2.3</version>
+			<scope>runtime</scope>
 		</dependency>
 
 		<!-- test dependencies -->
+
 		<dependency>
 			<groupId>org.junit.jupiter</groupId>
-			<artifactId>junit-jupiter-engine</artifactId>
-			<version>${junit.jupiter.version}</version>
+			<artifactId>junit-jupiter-api</artifactId>
+			<version>5.6.0</version>
 			<scope>test</scope>
 		</dependency>
 
 		<dependency>
-			<groupId>org.junit.vintage</groupId>
-			<artifactId>junit-vintage-engine</artifactId>
-			<version>${junit.jupiter.version}</version>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
```

Comment:
Here you also removed the (vintage) engine. According to the [JUnit 5 migration sample](https://github.com/junit-team/junit5-samples/tree/master/junit5-migration-maven/), one should keep it—but I haven't tested this.

Reply:
@beatngu13 I am discussing all of this together with @githubert - please view this PR rather as a draft PR, I will change the status accordingly